components/icu/Cargo.toml

```diff
@@ -63,11 +63,14 @@ version = "0.2"
 path = "../uniset"
 default-features = false
 
+[dependencies.fixed_decimal]
```

Comment:
Question: This isn't used in `icu` except in the one docs string. Does that one docs string compel us to add the direct dependency?

Reply:
Yes, unless we can find it as a reexport. But it's already being built indirectly so it's not a big deal.